Dockerfile

```diff
@@ -64,8 +67,10 @@ RUN if [ $EPEL_INSTALL -eq 1 ]; then yum install -y epel-release; fi\
          gdal-python \
          geos \
          httpd \
+         libffi-devel \
```

Comment:
devel packages will not be needed following build / install of cryptography python package. Recommend only installing the non devel packages in first yum install and then devel packages in yum install that will be rolled back.